daemon/daemon.go

```diff
@@ -322,6 +322,15 @@ func (d *Daemon) CleanCTEntries(e *endpoint.Endpoint, isCTLocal bool, ips []net.
 	endpointmanager.RmCTEntriesOf(!d.conf.IPv4Disabled, e, isCTLocal, ips, idsToRm)
 }
 
+// FlushCTEntries flushes the connection tracking of the given endpoint
+// where the given endpoint IPs' match the CT entry fields.
+// isCTLocal should bet set as true if the endpoint's CT table is either
+// local or not (if is not local then is assumed to be global).
+// Implementation of pkg/endpoint.Owner interface
```

Comment:
Thanks for adding this comment - makes tracking interface implementation easier!